packages/@aws-cdk/core/lib/cfn-parse.ts

```diff
@@ -294,6 +295,43 @@ export class CfnParser {
     return cfnValue;
   }
 
+  public handleAttributes(resource: CfnResource, resourceAttributes: any, logicalId: string): void {
+
+    const finder = this.options.finder;
+    const cfnParser = new CfnParser({
```

Comment:
Uhm. No reason to create a `cnfParser`, we are inside one :). So just call `this. parseCreationPolicy`, etc.